dev_docs/RELEASES.md

```diff
@@ -9,34 +9,17 @@ For patch releases, only the version on the existing major and minor version bra
 
 ## Feature Freeze
 
-* For patch releases, ensure all relevant backport PRs are merged.
-  We use backport labels on PRs and automation to ensure labels are set.
+* For **patch releases**:
+  * ensure all relevant backport PRs are merged. We use backport labels on PRs and automation to ensure labels are set.
 
```

Comment:
```suggestion
  * Ensure all relevant backport PRs are merged. Backport PRs are automatically opened based on `backport-X.Y` labels added to PRs opened against `main`.
  * Add release notes
    * Review the [changelogs/head](https://github.com/elastic/apm-server/tree/main/changelogs/head.asciidoc) file and move all relevant release notes from `head.asciidoc` to `release_version.asciidoc.
      * Minor version:
        Create a new changelog file from [`changelogs/head.asciidoc`](https://github.com/elastic/apm-server/blob/main/changelogs/head.asciidoc)
        If changes do not apply to the version being released, keep them in the `changelogs/head.asciidoc` file.
        Don't forget to `include` and link to the new file in [main changelog](https://github.com/elastic/apm-server/blob/main/CHANGELOG.asciidoc) file and the [release notes](https://github.com/elastic/apm-server/blob/main/docs/release-notes.asciidoc) file. [(Sample PR)](https://github.com/elastic/apm-server/pull/7956/files)
      * Patch version: Add a new section to existing release notes. ([Sample PR](https://github.com/elastic/apm-server/pull/8313/files))
    * Add `@elastic/obs-docs` as a reviewer.
```

Reply:
Thanks for the suggestion. I'm confused about lines 14-21 though: we have 2 automation that perform what is described there and we are documenting them on line 18-22 of the new RELEASES.md. May you share the rationale for adding that part again?

Reply:
Looks like it would've helped if I had actually read the entire file 😵‍💫

Reply:
So the manual process has been replaced with an automated one.

> I think it is missing docs release process. It used to be done by docs team but it is now our responsibility to do so. Can you figure out at which stage that should happen and update the docs, and give an example?

@carsonip I'm not sure what's missing then

Reply:
Thanks @bmorelli25! Then I will consider this conversation resolved (but won't do it in GitHub for future reference).

Reply:
Patch release automation does not include changelog changes. It has to be done in coordination with docs release manually. See https://github.com/elastic/dev/issues/2485 (private) and https://github.com/elastic/apm-server/pull/12680/files